Dockerfile

```
@@ -0,0 +1,35 @@
+From python:3.7-buster as basepython
+
+WORKDIR /opt/app-root/
+ENV PATH=/opt/app-root/bin:$PATH
+
+# Create an apt-root dir and set permissions, add a python
+# virtual environment and install pip.
+RUN /usr/local/bin/python -m venv /opt/app-root/ && \
+    /opt/app-root/bin/pip install -U pip && \
```

```suggestion
    /opt/app-root/bin/pip install -U pip wheel && \
```

including wheel here makes it easier when installing requirements